api/src/main/scala/dcos/metronome/api/Authorization.scala

```diff
@@ -75,6 +78,10 @@ trait Authorization extends RestController {
         case None           => Future.successful(notAuthenticated)
       }
     }
+
+    override def parser: BodyParser[AnyContent] = new BodyParsers.Default()
```

Comment:
Is this body parser the one we should be using?

Reply:
I injected the default body parser from play, I hope it's OK :)